packages/ove-asset-manager/src/OVE.Service.AssetManager/GlobalSuppressions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// ReSharper disable All
+
+// This file is used by Code Analysis to maintain SuppressMessage 
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given 
+// a specific target and scoped to a namespace, type, member, etc.
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Minor Code Smell", "S101:Types should be named in camel case", Justification = "Sonar has no custom dictionaries so cannot say OVE is ok", Scope = "type", Target = "~T:OVE.Service.AssetManager.Controllers.OVEAssetModelController")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Major Code Smell", "S4144:Methods should not have identical implementations", Justification = "Sonar Does not understand how Views work", Scope = "member", Target = "~M:OVE.Service.AssetManager.Controllers.OVEAssetModelController.GetRemovableView(System.String)~System.Threading.Tasks.Task{Microsoft.AspNetCore.Mvc.ActionResult{OVE.Service.AssetManager.Models.OVEAssetModel}}")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Minor Code Smell", "S1075:URIs should not be hardcoded", Justification = "its a single / ", Scope = "member", Target = "~M:OVE.Service.AssetManager.Domain.S3AssetFileOperations.ResolveFileUrl(OVE.Service.AssetManager.Models.OVEAssetModel)~System.String")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Info Code Smell", "S1135:Track uses of \"TODO\" tags", Justification = "justification given on each occurance")]

```

Comment:
@davidbirchwork, why is tracking TODO tags in code a false positive? I thought it makes sense to track these and warn the developer(s) that all of those needs to be removed before the code can be considered production-ready?

Reply:
rational was that unfortunately Sonar is not clever enough to pick up and report the rational from the code for each occurrence. 

I can remove this suppression if preferred.